evennia/utils/create.py

```diff
@@ -501,6 +501,8 @@ def create_account(
         report_to (Object): An object with a msg() method to report
             errors to. If not given, errors will be logged.
 
+    Returns:
+        (object) The acount object of the new account.
```

Comment:
This has a typo in it, and is also not written on the correct form (see the style guide)

    Returns: 
        account (Account): The account object of the new account.